### ISSUE TITLE
ODROID XU4 cannot boot with eMMC

### ISSUE THREAD

**Issue by hutzero** [libretro/Lakka-LibreELEC#224]
I have tried on nightly build already. It still not fix. 
May this souce below could help you to find root of cause.

https://github.com/nadenislamarre/batocera.linux/tree/master/board/recalbox/xu4

**Comment by Kivutar** [comment]
In fact we know the cause. The adapter eMMC2SD prevents us to access the first sector of the eMMC where we're supposed to flash u-boot.

Luckily, the eMMC you buy from hardkernel already have an u-boot flashed on the first sector. So we could generate a special image with an offset of 1, and it would then work for the eMMC (if it comes from hardkernel).

But it would stop working for SD.

**Comment by hutzero** [comment]
Yes, I brought it from hardkernel. I can use batocera(recalbox) with both of eMMC and microSD in the same image. But if you cannot build Lakka in the same image like them, please split it to 2 versions. (I was ever try the previous special build from you in the forum, but not working)

**Comment by Kivutar** [comment]
If they did it, then we can also do it. In fact, it was already the case on old versions of Lakka, before the LibreELEC switch. I should look into that.

**Comment by Kivutar** [comment]
Hardkernel is going to send me two new eMMC and I will update their kernel and u-boot for XU3/4

**Comment by hutzero** [comment]
Thank you very much for your hard work in this issue.  I will become a Patreon to support every month, if this issue has fixed. I promise.

**Comment by Kivutar** [comment]
I found a fix
